pom.xml

```diff
@@ -57,7 +57,7 @@
     <jee.port>3333</jee.port>
     <refine.data>/tmp/refine</refine.data>
     <surefire.version>2.22.2</surefire.version>
-    <surefireArgs></surefireArgs>
+    <surefireArgs>-Dfile.encoding=cp1252</surefireArgs>
```

Comment:
I guess we should ideally be able to apply this only for specific tests, no? I am not sure how that could be done, though. I am trying to think about the risks we are taking by switching to this encoding everywhere. Intuitively it should help us surface more bugs rather than hide ones which would only happen if the system encoding is UTF-8, so it is probably fine?

Reply:
That was my thinking. We assume/need UTF-8 generally, but our most popular platform doesn't use that as its default encoding, so neither should our tests. I think all tests should be run with this setting.